k8s_helm_charts/zdm/values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for zdm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: zdmproxy
+
+origin_username:
+origin_password:
+target_username:
+target_password:
+
+proxy:
+  count: 3
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 512Mi
+    requests:
+      cpu: 1000m
+      memory: 512Mi
+  image:
+    repository: datastax/zdm-proxy
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
```

Comment:
empty string tag doesn't make a lot of sense for the default, I think? you really *do not* want `latest` for anything more than a short lived debugging container, in k8s

Reply:
So in deployment we have the following reference for the proxy image:
`    spec:
      containers:
      - image: "{{ $.Values.proxy.image.repository }}:{{ $.Values.proxy.image.tag | default $.Chart.AppVersion }}"`

Reply:
👍